prometheus.lua

```diff
@@ -448,6 +481,37 @@ function Prometheus:inc(name, label_names, label_values, value)
   self:log_error_kv(key, value, err)
 end
 
+-- Decrement a given gauge by `value`.
+--
+-- Args:
+--   name: (string) short metric name without any labels.
+--   label_names: (array) a list of label keys.
+--   label_values: (array) a list of label values.
+--   value: (number) value to add. Optional, defaults to 1.
+function Prometheus:dec(name, label_names, label_values, value)
```

Comment:
This is almost exact copy of `Prometheus:inc`. What I would suggest instead is:
- make `Prometheus:inc` require a value (instead of assuming it's `1` when no value us supplied).
- move "value should be non-negative" check to `Counter:inc` (no such check is necessary for gauges).
- call `Prometheus:inc` with a negative value when you need to decrease the value of the gauge.

Reply:
Thanks for you suggestions. I think that will be much simpler.